tests/acceptance/addon-smoke-test-slow.js

```diff
@@ -34,23 +34,18 @@ describe('Acceptance: addon-smoke-test', function() {
     });
   });
 
-  after(function() {
-    return teardownTestTargets();
-  });
+  after(teardownTestTargets);
 
   beforeEach(function() {
-    return linkDependencies(addonName).then(function(result) {
-      addonRoot = result;
-    });
+    addonRoot = linkDependencies(addonName);
   });
 
   afterEach(function() {
     // Cleans up a folder set up on the other side of a symlink.
     fs.remove(path.join(addonRoot, 'node_modules', 'developing-addon'));
 
-    return cleanupRun(addonName).then(function() {
-      expect(dir(addonRoot)).to.not.exist;
-    });
+    cleanupRun(addonName);
```

Comment:
In distance past this was sync, in the not-so-distant past this was made async due to windows being unable to consistently synchronously remove large trees of files.

As such I am a tad nervous to change this, but if demonstrated that windows doesn't suffer from this issue. My nerves will be settled.